frontend/src/components/v1/RelatedWalletQuickLinks.css

```
@@ -0,0 +1,174 @@
+.related-wallet-links {
+  --wallet-bg: color-mix(in oklab, var(--surface-elevated, rgba(255, 255, 255, 0.08)) 92%, transparent);
+  --wallet-border: color-mix(in oklab, var(--glass-border, rgba(255, 255, 255, 0.16)) 86%, transparent);
+  --wallet-text: var(--text-main, #f5f7fb);
+  --wallet-text-muted: var(--text-secondary, #a8b5c8);
+  --wallet-hover-bg: color-mix(in oklab, var(--surface-elevated, rgba(255, 255, 255, 0.08)) 85%, transparent);
+  --wallet-active-bg: color-mix(in oklab, var(--accent, #7de2d1) 20%, rgba(255, 255, 255, 0.04));
+  --wallet-active-border: color-mix(in oklab, var(--accent, #7de2d1) 45%, rgba(255, 255, 255, 0.16));
+  display: block;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)